Paralayout/GeometryAdditions.swift

```diff
@@ -89,31 +89,37 @@ extension CGRect {
     public func inset(left: CGFloat = 0.0, top: CGFloat = 0.0, right: CGFloat = 0.0, bottom: CGFloat = 0.0) -> CGRect {
         return inset(by: UIEdgeInsets(top: top, left: left, bottom: bottom, right: right))
     }
-    
-    /// Outsets the rect, if necessary, to snap to the nearest pixel at the specified scale.
-    /// - parameter scaleFactor: The pixel scale to use, e.g. a UIScreen, UIView, or explicit value (pass `0` to *not*
-    /// snap to pixel).
-    /// - returns: A new rect with pixel-aligned boundaries, enclosing the original rect.
-    public func expandToPixel(_ scaleFactor: ScaleFactorProviding) -> CGRect {
-        return CGRect(
-            left: minX.floorToPixel(in: scaleFactor),
-            top: minY.floorToPixel(in: scaleFactor),
-            right: maxX.ceilToPixel(in: scaleFactor),
-            bottom: maxY.ceilToPixel(in: scaleFactor)
-        )
+
+    /// Insets the rect equally on all sides.
+    /// - parameter inset: The amount by which to contract each edge of the receiver.
+    /// - returns: A new rect with the inset applied.
+    public func inset(allSides inset: CGFloat) -> CGRect {
```

Comment:
All of these feel like they should be `insetBy` for consistency. If I had to guess they went with `insetBy` over just `inset` so that it's clear it's returning the new rect and not the inset itself. But that's a guess; either way we should be consistent.

Reply:
UIKit defines `insetBy(dx:dy:)` and `inset(by:)`, so I agree adding "by" probably makes sense for consistency. Following that pattern, this could be:

1. `insetBy(allSides:)`
2. `inset(allSidesBy:)`
3. `insetAllSides(by:)`

I can see an argument for any of these. What's your preference?

Reply:
I think `insetAllSides(by:)` feels most idiomatic, but don't feel too strongly.

Reply:
Works for me. Updated in 8669711 and fdb1c33.